android/src/main/java/com/swmansion/rnscreens/Screen.java

```diff
@@ -151,7 +152,22 @@ public void setTransitioning(boolean transitioning) {
       return;
     }
     mTransitioning = transitioning;
-    super.setLayerType(transitioning ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_NONE, null);
+    boolean isWebViewInScreen = hasWebView(this);
+    super.setLayerType(transitioning && !isWebViewInScreen ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_NONE, null);
```

Comment:
This line isn't consistent with the PR description. You say that we don't change the layer type in case there is a webview, but it appears that we change it to NONE. Should we avoid updating layer type altogether as stated in the description?

Reply:
We could not change the layer type, but if it is already `HARDWARE` for some reason, it will probably lead to crash. I can add a check for if the layer is not `HARDWARE` and, if so, just do nothing when there is a `WebView`.